giles-eco/src/main/webapp/WEB-INF/tiles/skeleton.jsp

```diff
@@ -111,6 +111,9 @@
           	<li role="presentation">
           		<a href="<c:url value="/admin/apps" />" >Apps</a>
           	</li>
+          	<li role="presentation">
+          		<a href="<c:url value="/admin/kafka/listeners" />">Kafka</a>
+          	</li>
```

Comment:
It get's a little crowded up there in the main menu. Let's move this as a new entry under "Requests" as "Start/Stop Kafka Listeners".

Reply:
Made the changes as described above and requested for the review.